modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -525,6 +548,17 @@ else if ((nestedFieldsDepth > 0) &&
 					GetterUtil.getString(
 						values.get(objectRelationshipERCFieldName)));
 			}
+			else if (Objects.equals(
+						objectField.getBusinessType(),
+						ObjectFieldConstants.BUSINESS_TYPE_PRECISION_DECIMAL)) {
+
+				DecimalFormat defaultDecimalFormat = _getDecimalFormat(
+					LocaleUtil.getDefault());
```

Comment:
The locale should be from the dtoConverterContext.getLocale(). Could you check if it fits with the ticket scenario?

Reply:
I updated source based on your suggest. Thank you!